library/src/main/java/com/cube/storm/ui/model/descriptor/TabbedPageDescriptor.java

```diff
@@ -18,7 +19,7 @@
  */
 @NoArgsConstructor @AllArgsConstructor
 @Accessors(chain = true) @Data @EqualsAndHashCode(callSuper=false)
-public class TabbedPageDescriptor extends PageDescriptor
+public class TabbedPageDescriptor extends PageDescriptor implements Parcelable
```

Comment:
I am not sure about this class implementing Parcelable. It is good practice, but as things stand `tabBarItem` is not included and so if you serialize and then deserialize it then `tabBarItem` will be null... What was the reason to make it Parcelable?

Reply:
I saw the Parcelable declaration incomplete, so I decided to implement it because it was used in a lot of places. However, it is not used this feature, so I'm going to leave it as it was 😃